lib/streamlit/Report.py

```diff
@@ -64,13 +64,23 @@ def __init__(self, script_path, argv):
         argv : list of str
             Command-line arguments to run the script with.
 
+        command_line : string
+            Command line as input by the user
+
         """
-        basename = os.path.basename(script_path)
 
         self.script_path = os.path.abspath(script_path)
         self.script_folder = os.path.dirname(self.script_path)
         self.argv = argv
-        self.name = os.path.splitext(basename)[0]
+        self.command_line = command_line
+
+        if len(command_line.split()) > 2:
```

Comment:
I don't understand what this `if` block is doing.

Reply:
yeah,... removed...